essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/ZoomRoom/ZoomRoom.cs

```diff
@@ -2594,6 +2595,7 @@ public void LinkZoomRoomToApi(BasicTriList trilist, ZoomRoomJoinMap joinMap)
 				if (args.LoginAttemptFailed)
 				{
 					// login attempt failed
+					_meetingPasswordRequired = false;

```

Comment:
We should remove this line.  It's already handled inside OnPasswordRequired helper method

Reply:
Removed as recommended.